daemon/mgr/cri.go

```diff
@@ -2,11 +2,37 @@ package mgr
 
 import (
 	"fmt"
+	"strings"
+
+	pouchtypes "github.com/alibaba/pouch/apis/types"
 
 	"golang.org/x/net/context"
```

Comment:
I don't like the "context" package, I hope to use standard "context" to replace it.

We maybe need to find a way to do it.  

WDYT @allencloud